src/connectedk8s/azext_connectedk8s/custom.py

```diff
@@ -108,6 +108,11 @@ def create_connectedk8s(cmd, client, resource_group_name, cluster_name, https_pr
     }
     telemetry.add_extension_event('connectedk8s', kubernetes_properties)
 
+    # Checking if it is an AKS cluster
+    is_aks_cluster = check_aks_cluster(kube_config, kube_context)
+    if is_aks_cluster:
+        logger.warning("The cluster being onboarded is an AKS cluster. While Arc onboarding an AKS cluster is possible, it's not necessary. Learn more at {}.".format(" https://go.microsoft.com/fwlink/?linkid=2144200"))
```

Comment:
Slight change - Use "The cluster you are trying to connect to Azure Arc is an Azure Kubernetes Service (AKS) cluster" for the first sentence

Reply:
Okay, will change.